var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -17,7 +17,7 @@ class Ucx(AutotoolsPackage, CudaPackage):
     maintainers = ['hppritcha']
 
     # Development
-    version('1.9-dev', branch='v1.9.x')
+    version('1.10-dev', branch='v1.10.x')
```

Comment:
If you name this `1.10-develop` it should sort correctly. Also, should we keep the old version too?